image_lists/global.yml

```diff
@@ -3,12 +3,6 @@ hourly:
   variables:
     1ref:
       - 1000m
-    acpcp:
-      - sfc
-    acsnod:
-      - sfc
-    acsnw:
-      - sfc
```

Comment:
Because of the presence of another total acc variable, the renaming has trouble, so these will be left out for now until a fix is put in.